libc/string/string_cmp.c

```diff
@@ -0,0 +1,439 @@
+/*
+ * Phoenix-RTOS
+ *
+ * POSIX.1-2017 standard library functions tests
+ * HEADER:
+ *    - string.h
+ * TESTED:
+ *    - memcmp()
+ *    - strcmp()
+ *    - strncmp()
+ *    - strcoll()
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Damian Modzelewski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+
+#include <stdio.h>
+#include <errno.h>
+#include <string.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <unity_fixture.h>
+
+#include "testdata.h"
+
+
+#define BUFF_SIZE 129
+#define BIG_SIZE  1024
+
+static char empty[BUFF_SIZE] = { 0 },
```

Comment:
It's y default initialized to 0, when it's global buffer.

```suggestion
static char empty[BUFF_SIZE];
```